test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/DatabaseHandlerTests.cs

```diff
@@ -260,6 +260,50 @@ public void GetAzureMaxSizesTest()
             }
         }
 
+        [Test]
+        /// This test validates the newly created database properties and verifies with some default values
+        public async Task VerifyDatabasePropertiesTest()
+        {
+            // setup, drop database if exists.
+            var connectionResult = await LiveConnectionHelper.InitLiveConnectionInfoAsync("master", serverType: TestServerType.OnPrem);
+            using (SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connectionResult.ConnectionInfo))
+            {
+                var server = new Server(new ServerConnection(sqlConn));
+
+                var testDatabase = ObjectManagementTestUtils.GetTestDatabaseInfo();
```

Comment:
Maybe `testDatabaseInfo` instead.

But also I'd be careful with trying to use actual "default" values like 4 users and size. Those values are not something we control so it makes this test inherently unstable. I think it's fine for this test just to verify that we have some value for the properties like that which we can't control.